pkg/gcp/gcs/gcs.go

```diff
@@ -106,9 +106,8 @@ func bucketCopy(src, dst string, opts *Options) error {
 
 	args = append(args, src, dst)
 
-	cpErr := command.Execute(gcp.GSUtilExecutable, args...)
-	if cpErr != nil {
-		return errors.Wrap(cpErr, "gcs copy")
+	if err := command.Execute(gcp.GSUtilExecutable, args...); err != nil {
+		return errors.Wrap(err, "gcs copy")
```

Comment:
Should we evaluate using `cloud.google.com/go/storage` instead of gsutil here ?

Reply:
I see this as an long term effort. Right now we use the native libraries only in the push build ACL access check. I think we can migrate to the go/storage library, especially since bazel is gone and we can upgrade the reps. WDYT?

Reply:
Yes, we could start looking into that once the anagoification is complete and we start optimizing more that converting :+1: